include/sys/fs/zfs.h

```diff
@@ -741,6 +744,24 @@ typedef enum zpool_errata {
 	ZPOOL_ERRATA_ZOL_2094_ASYNC_DESTROY,
 } zpool_errata_t;
 
+#ifndef BIT /* Include BIT() for userspace */
+#define	BIT(n)	(1ULL << (n))
+#endif
+
+enum vdev_stat_flags {VS_DEFAULT = 0, VS_LATENCY = 1, VS_QUEUES = 2,
+		VS_HISTO = 3, VS_FLAGS_COUNT /* always last element */};
+
+/* Mask versions of all our flags */
+#define	VS_DEFAULT_M	BIT(VS_DEFAULT)
+#define	VS_LATENCY_M	BIT(VS_LATENCY)
+#define	VS_QUEUES_M	BIT(VS_QUEUES)
+#define	VS_HISTO_M	BIT(VS_HISTO)
```

Comment:
Good point, I can see us adding a request size histogram at some point.  I want to be careful though, since the macros are used a lot, and the longer names will push code towards the 80-char limit.  At the least, I'll rename VS_HISTO to something more specific, since we'll probably of adding more histos.